Homework_6/task6.3/main.cpp

```diff
@@ -0,0 +1,95 @@
+#include <iostream>
+#include <fstream>
+#include "list.h"
+#include "phonenumbers.h"
+
+using namespace std;
+
+enum Command {
+    exit, add, find1, find2, save
+};
+
+Command setCommand(int tmp)
+{
+    return static_cast<Command>(tmp);
+}
+
+int main()
+{
+    ifstream fin;
+    fin.open("phone_numbers.txt");
+    List *phonebook = createList();
+    Person *newPerson = createPerson();
+    int amount = 0;
+    fin >> amount;
+    char curSymbol = 0;
+    fin.get(curSymbol);
+    for (int i = 0; i < amount; ++i)
+    {
+        input(fin, newPerson->phone);
+        input(fin, newPerson->name);
+        pushBack(phonebook, *newPerson);
+    }
+    fin.close();
+
+    ofstream fout;
+    fout.open("phone_numbers.txt");
+    int tmp = 1;
+    Command way = setCommand(tmp);
+    char name[maxSize];
+    char phone[maxSize];
+    ListElement *searched = nullptr;
+    while (way != exit)
```

Comment:
main.cpp:42:16: error: no match for ‘operator!=’ (operand types are ‘Command’ and ‘void(int) throw ()’)